docs/getting-started-on-snowplow-open-source/quick-start-aws/query-your-data/query-your-data-in-databricks/index.md

```diff
@@ -0,0 +1,38 @@
+---
+title: "Query your data in Databricks"
+date: "2022-05-10"
+sidebar_position: 400
+---
+
+If you have created Databricks resources by following [this guide](/docs/getting-started-on-snowplow-open-source/quick-start-aws/index.md) and deployed the pipeline using the `default` or `secure` option, you will have a Databricks database where all of your enriched data is stored.
```

Comment:
OK for now, but we will need to restructure this. I’d like to move “how to query” to a separate section that’s the same for all users regardless of how they installed stuff. So for the quick start guide we need to pare this down to just “how to connect” (probably a single page with tabs, rather than multiple pages) and link to the querying doc for any further info. Again, trying to separate installation from usage.